test/controllers/EmailControllerSpec.scala

```diff
@@ -39,26 +47,50 @@ class EmailControllerSpec extends SpecBase {
     }
 
     "return unverified email response" in new Setup {
+
+      when[Future[EmailUnverifiedResponse]](
+        mockHttpClient.GET(ArgumentMatchers.endsWith("/subscriptions/unverified-email-display"), any, any)
+        (any, any, any))
+        .thenReturn(Future.successful(response))
+
       running(app) {
         val request = fakeRequest(GET, routes.EmailController.showUnverified().url)
         val result = route(app, request).value
         status(result) shouldBe OK
       }
     }
+
+    "return undeliverable email response" in {
```

Comment:
Better to rename the test description to make it more meaningful like below as its showing the email undeliverable view as part of  this method

"display undeliverable email page"